modules/primer-avatars/lib/avatar-stack.scss

```diff
@@ -43,3 +43,114 @@
     }
   }
 }
```

Comment:
We definitely could (and should!), but I don't think that's feasible by tomorrow.

Reply:
Hey @califa we need to update all the instances of avatar in GitHub for this to ship. As I understand it, this pattern is to _replace_ the current avatar stack, which means we need to ensure this pattern works everywhere it's being used. The best way to test this is to update the locations where this is being used. If you can't do that in time for v10 release, or you need help with that, then we'll need to discuss whether to pull it out of v10 or delay it by a few days. I'd prefer to get it into v10 because I think these changes will require a major release and I don't really want to do v11 straight after v10.

For clarity on timeline, we want all v10 pr's merged in eod tomorrow, so that we can do a naming change on Monday and test, and then intend to ship Monday or Tuesday depending on how long it takes to test.

Reply:
@broccolini 👍 talked to @sophshep. I'm spending all day on this today.